lib/profile/sections/contact.dart

```diff
@@ -4,10 +4,17 @@ import 'package:url_launcher/url_launcher.dart';
 import 'package:flutter/material.dart';
 import 'package:mentorApp/constants/profile_constants.dart';
 import './header.dart';
+import '../../chat/chat_messenger.dart';
+
  
 class ProfileContact extends StatelessWidget {  
   final Map contact;
-  ProfileContact({Key key, @required this.contact}) : super(key: key);
+  final String fullName;
+  ProfileContact({Key key, @required this.contact, @required this.fullName}) : super(key: key);
+
+  final IconData chat = const IconData(62459,
```

Comment:
If you wanted to, you could quickly come in here and put `62459` on a separate line for code style consistency.